sway/input/cursor.c

```diff
@@ -313,6 +314,13 @@ void cursor_unhide(struct sway_cursor *cursor) {
 		cursor->image = NULL;
 		cursor_set_image(cursor, image, cursor->image_client);
 	}
+	struct sway_node *focus = seat_get_focus(cursor->seat);
```

Comment:
I think this should be replaced with `cursor_rebase(cursor)` so that it handles layer surfaces as well

Reply:
Nice catch, this fixed the input bug I was experiencing.